Hearthstone Deck Tracker/GameEventHandler.cs

```diff
@@ -1362,4 +1362,4 @@ public void HandleOpponentDeckDiscard(Entity entity, string cardId, int turn)
 
 		#endregion IGameHandlerImplementation
 	}
-}
\ No newline at end of file
+}
```

Comment:
Not sure why it did that, I did it on the GH text editor as it was a short edit.